examples/browser-webrtc/client/Cargo.toml

```diff
@@ -21,14 +24,15 @@ libp2p = { path = "../../../libp2p", features = [
 ] }
 libp2p-webrtc-websys = { workspace = true }
 log = "0.4"
-wasm-bindgen = "0.2.87"
+wasm-bindgen = "0.2.84"
 wasm-bindgen-futures = "0.4.37"
-web-sys = { version = "0.3.64", features = [
-    "Response",
-    "Window",
-    'Performance',
-    'PerformanceTiming',
-] }
 
-[features]
-default = ["console_log", "fern"]
+# The `console_error_panic_hook` crate provides better debugging of panics by
+# logging them with `console.error`. This is great for development, but requires
+# all the `std::fmt` and `std::panicking` infrastructure, so isn't great for
+# code size when deploying.
```

Comment:
Do we actually care for an example? I'd rather prefer low-complexity :)

Reply:
I am working on simplifying the example and reducing the additional dependencies (to zero, I think). But I do find that newcomers to the libp2p ecosystem appreciate the examples for self directed guidance and inspiration (I certainly did!) so I think it's valuable to keep around.